.github/workflows/quic_interop_runner.yml

```diff
@@ -0,0 +1,34 @@
+name: QUIC Interop Runner
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: [ubuntu-20.04]
+
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      continue-on-error: false
+
+    - name: Build and deploy
+      run: |
+        if [ "${{ github.event_name }}" != "push" ]
+        then
+          export TAG=canary
```

Comment:
Could you clarify the intent behind this, please? When are they kicked, why can there share the same name?

Reply:
I think all we need is `latest`, which is used by QUIC Interop Runner.

So my assumptions are like this:

* We need to push an image only when the workflow is triggered by an `on.push` event
* The other event, i.e., `on.pull_request`, are just for testing, so its artifacts can be discarded
* Thus we have two ways in `on.pull_request` events
  * (1) push images with an arbitrary tag name but `latest`
  * (2) skip `docker push` when it's triggered by `on.pull_request`

I've picked (1) for testing so far.

Reply:
Thank you for elaborating. I think I agree that the two options that you point out are probably the ones that makes the most sense. 

I tend to prefer option 2, because it reduces the burden we put on docker.com (I recall them recently limiting downloads), and also because docker images should only contain executable being built from source code that we trust - PR from an arbitrary person does not meet that criteria.

WDYT?

Reply:
Agreed. Pushing images is not necessary and even potentially harmful. 

Skipping `docker push` on pull-requests: f3dc54728